akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -56,6 +56,11 @@ object Flow {
   /**
    * Creates a `Flow` from a `Sink` and a `Source` where the Flow's input
    * will be sent to the Sink and the Flow's output will come from the Source.
+   *
+   * The Sink and Source sides of a flow constructed like this have completion
+   * of either side independent. So if the Sink receives a completion signal,
```

Comment:
Yea. This sounds better. Thanks. I changed your original proposal since it had too much "quotes" around words. :)